GoogleSignIn.podspec

```diff
@@ -33,9 +33,9 @@ The Google Sign-In SDK allows users to sign in with their Google account from th
   ]
   s.ios.framework = 'UIKit'
   s.osx.framework = 'AppKit'
-  s.dependency 'AppAuth', '~> 1.6'
-  s.dependency 'GTMAppAuth', '~> 4.0'
-  s.dependency 'GTMSessionFetcher/Core', '>= 1.1', '< 4.0'
+  s.dependency 'AppAuth', '~> 1.7'
+  s.dependency 'GTMAppAuth', '~> 4.1'
+  s.dependency 'GTMSessionFetcher/Core', '~> 3.3'
```

Comment:
note: opted to switch over to the optimistic operator since bumping the min version required to 3.3 would make `'>= 3.3', '< 4.0'` equivalent to `'~> 3.3'`